assets/blocks/button/settings-button.js

```diff
@@ -45,14 +45,14 @@ export const BorderPanel = ( { borderRadius, setAttributes } ) => {
  */
 export const ButtonBlockSettings = ( props ) => {
 	const { attributes, setAttributes } = props;
-	const { borderRadius, align } = attributes;
+	const { borderRadius, textAlign } = attributes;
 	return (
 		<>
 			<BlockControls>
 				<AlignmentToolbar
-					value={ align }
-					onChange={ ( nextAlign ) => {
-						setAttributes( { align: nextAlign } );
+					value={ textAlign }
```

Comment:
Can we use `getThemeSupports` to hide this setting if the theme doesn't support wide alignment?

Reply:
TIL
Great catch! Thank you!